Sources/SnapshotTesting/Strategy/Codable.swift

```diff
@@ -9,8 +9,26 @@ extension Strategy where A: Encodable {
   }
 
   public static func json(_ encoder: JSONEncoder) -> Strategy<A, String> {
-    return Strategy.lines.pullback { encodable in
+    var strategy = Strategy.lines.pullback { (encodable: A) in
       try! String(decoding: encoder.encode(encodable), as: UTF8.self)
     }
+    strategy.pathExtension = "json"
+    return strategy
   }
+
+  #if !os(Linux)
```

Comment:
Out of curiosity - are you excluding linux because PropertyListEncoder is not available? I’ve worked around this in my project by pulling in a file from the stdlib. See 
https://finestructure.co/blog/2018/10/7/make-propertylistdecoder-available-on-linux